tests/incentivization/test_rpc_codec.nim

```diff
@@ -0,0 +1,32 @@
+import
+  std/options,
+  std/strscans,
+  testutils/unittests,
+  chronicles,
+  chronos,
+  libp2p/crypto/crypto
+
+import
+  ../../../waku/incentivization/rpc,
+  ../../../waku/incentivization/rpc_codec
+
+
+suite "Waku Incentivization Eligibility Codec":
+
+    asyncTest "encode eligibility proof":
+      var byteSequence: seq[byte] = @[1, 2, 3]
```

Comment:
Nitpick: I would add more bytes
```suggestion
      var byteSequence = @[0..31, byte]
```

Reply:
Got an error on the modified line, what am I doing wrong?

```
Error: type mismatch: got 'typedesc[byte]' for 'byte' but expected 'HSlice[system.int, system.int]'

```

Reply:
ah sorry that's a Rust thing. In Nim you can't initialize an array with a range AFAIK.

Reply:
OK, I'll then add more bytes manually (not 32, perhaps 8).